app/ts/components/pages/WebsiteAccess.tsx

```diff
@@ -0,0 +1,561 @@
+	}
+
+	if (addressIcons.length < 1) return <></>
+
+	return (
+		<div style = { { paddingLeft: '2.5rem', display: 'grid', gridAutoFlow: 'column', gridAutoColumns: 'min-content', columnGap: '0.5rem', alignItems: 'center', paddingBottom: '0.5rem' } }>
+			{ addressIcons.slice(0, MAX_ADDRESS_ICONS) }
+			{ addressIcons.length > MAX_ADDRESS_ICONS ? <div style = { { fontSize: '0.75rem', lineHeight: 1, padding: '0.125rem 0.1875rem', backgroundColor: 'darkgoldenrod', color: 'white', borderRadius: 2, fontWeight: 600, border: '1px solid goldenrod' } }>+{ addressIcons.length - MAX_ADDRESS_ICONS }</div> : <></> }
+			{ addressIcons.length > MAX_ADDRESS_ICONS ? (
+				<div style = { { color: 'darkgoldenrod', fontSize: '0.75rem', whiteSpace: 'nowrap' } }> addresses</div>
+			) : <></> }
+		</div>
+	)
+}
+
+const WebsiteSettingsDetail = () => {
+	const { websiteAccessList, selectedDomain } = useWebsiteAccess()
+	const dialogRef = useRef<HTMLDialogElement>(null)
+
+	const websiteAccess = useComputed(() => websiteAccessList.value.find(access => access.website.websiteOrigin === selectedDomain.value)!)
+
+	const closeDetails = () => { selectedDomain.value = undefined }
+
+	useSignalEffect(() => {
+		const dialogElement = dialogRef.current
+		if (!websiteAccess.value || dialogElement === null) return
+		dialogElement.showModal()
+	})
+
+	useEffect(() => {
+		const dialogElement = dialogRef.current
+		if (!dialogElement) return
+		dialogElement.addEventListener('close', closeDetails)
+		return () => dialogElement.removeEventListener('close', closeDetails)
+	}, [dialogRef.current])
+
+	return (
+		<dialog ref = { dialogRef } class = 'access-details'>
+			<form method = 'dialog' class = 'layout' onSubmit = { closeDetails }>
+				<header style = { { paddingBlock: '1rem' } }>
+					<button type = 'submit' class = 'btn btn--ghost' style = { { fontSize: '0.875rem', paddingInline: '0.5rem', paddingBlock: '0.125rem' } } autoFocus>&larr; Show website access list</button>
+					{ !websiteAccess.value ? <></> : (
+						<div class = 'flexy flexy-sm' style = { { '--gap-x': '1rem', flex: 1 } }>
+							<figure><img width = '34' height = '34' src = { websiteAccess.value.website.icon } /></figure>
+							<div style = { { flex: 1 } }>
+								<h2 class = 'truncate' style = { { fontSize: 'clamp(1.25rem,2vw,2rem)', fontWeight: 600, color: 'var(--text-color)' } }>{ websiteAccess.value.website.title }</h2>
+								<p><span class = 'truncate' style = { { flex: 1, lineHeight: 1, color: 'var(--disabled-text-color)', direction: 'rtl', textAlign: 'left' } }>&lrm;{ websiteAccess.value.website.websiteOrigin }</span></p>
+							</div>
+						</div>)
+					}
+				</header>
+				{ websiteAccess.value ? (
+					<article>
+						<AddressAccessList websiteAccess = { websiteAccess } />
+						<AdvancedSettings websiteAccess = { websiteAccess } />
+					</article>
+				) : <NoAccessPrompt />
+				}
+			</form>
+
+		</dialog>
+	)
+}
+
+const NoAccessPrompt = () => {
```

Comment:
if website has no access. Its denied. You need to enable the site to grant access. Going to website wont help as the access is denied from the website